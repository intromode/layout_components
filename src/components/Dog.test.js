import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog.js';

describe('testing header component', () => {
  it('matches a screenshot', () => {
    const wrapper = shallow(<Dog />);
    expect(wrapper).toMatchSnapshot();
  });
});
