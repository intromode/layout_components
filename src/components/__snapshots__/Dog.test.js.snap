// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing header component matches a screenshot 1`] = `ShallowWrapper {}`;
